src/organisms/homeProductList/HomeProductList.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import VerticalWrapper from "../../atoms/verticalWrapper";
+import HorizontalWrapper from "../../atoms/horizontalWrapper";
+import Heading from "../../atoms/heading";
+import Text from "../../atoms/text";
+import Dropdown from "../../atoms/dropdown/Dropdown";
+
+import ProductView from "../../molecules/productView/ProductView";
+
+import { dropDownOptions } from "./homeProductList.data";
+
+const RenderProductList = React.memo(({ sortOrder = "NONE", categoryItems }) => {
+  const getSortedItems = React.useCallback(() => {
+    if (sortOrder === "NONE") return categoryItems?.sort((a, b) => a.order - b.order);
```

Comment:
Create constants for all these sort order values so if in future you want to change it, you don't have to change at all places